packages/jellyfish-api-core/__tests__/category/blockchain/getTxOutSetInfo.test.ts

```diff
@@ -0,0 +1,32 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+
+describe('TxOutSetInfo', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.generate(1)
```

Comment:
```suggestion
    await container.waitForBlockHeight(1)
```

Using this function might be more deliberate.